sources/packages/web/src/components/institutions/modals/EditInstitutionUserModal.vue

```diff
@@ -1,293 +1,159 @@
-                <strong>
-                  Selected:<span v-if="isAdmin"> Yes </span
-                  ><span v-else> No </span>
-                </strong>
-              </span>
-              <InputSwitch v-model="isAdmin" />
-            </v-col>
-            <v-col v-if="isAdmin">
-              <span class="form-text text-muted mb-2"
-                ><strong>Select Admin Role</strong></span
-              >
-              <!-- TODO: remove inline styles when moving to Vuetify component. -->
-              <Dropdown
-                v-model="selectedAdminRole"
-                :options="adminRoles"
-                :style="{ width: '20vw' }"
-                optionLabel="name"
-              />
-            </v-col>
-          </v-row>
-          <span v-if="!isAdmin">
-            <v-divider></v-divider>
-            <h4 class="color-blue">Location Based Access</h4>
-            <v-row
-              v-if="institutionLocationList && !institutionLocationList.length"
-              ><Message :closable="false"
-                >Institution locations need to be added first inorder to add a
-                non-admin users to institution</Message
-              >
-            </v-row>
-            <span v-else>
-              <v-row
-                ><v-col>
-                  <span class="form-text text-muted mb-2">
-                    <strong>Locations</strong>
-                  </span> </v-col
-                ><v-col>
-                  <span class="form-text text-muted mb-2">
-                    <strong>User Type</strong>
-                  </span>
-                </v-col>
-              </v-row>
-              <v-row v-for="location in institutionLocationList" :key="location"
-                ><v-col>
-                  <span>{{ location?.name }} </span><br />
-                  <span>
-                    <span>{{ location?.data?.address?.addressLine1 }}, </span>
-                    <span>{{ location?.data?.address?.city }}, </span>
-                    <span>{{ location?.data?.address?.provinceState }}</span>
-                    <br />
-                  </span>
-                </v-col>
-                <v-col>
-                  <Dropdown
-                    v-model="location.userType"
-                    :options="userType"
-                    optionLabel="name"
-                    placeholder="Select a User Role"
-                    :style="{ width: '25vw' }"
-                    :class="invalidUserType ? 'p-invalid' : ''"
-                  />
-                </v-col>
-              </v-row>
-            </span>
-          </span>
-        </form>
-      </v-container>
-    </v-sheet>
+    <template #content>
+      <institution-user-management
+        ref="institutionUserManagement"
+        :initialData="initialData"
+      >
+        <template #user-name>
+          <v-text-field
+            v-model="userInfo.displayName"
+            disabled
+            style="min-width: 300px"
```

Comment:
can we avoid inline styles

Reply:
It was a temporary fix, I moved to scoped class style with a comment for now.

Reply:
Moved.